litex_boards/platforms/sds1104xe.py

```diff
@@ -66,36 +66,50 @@
         IOStandard("LVCMOS33"),
     ),
 
+    # Scope things
+    ("offsetmux", 0,
+        Subsignal("S",       Pins("U14 Y18 AA18")),
+        Subsignal("nE",      Pins("U19")),
+        IOStandard("LVCMOS15"),
+    ),
+
+    ("offsetdac", 0,
+        Subsignal("SCLK",       Pins("H15")),
+        Subsignal("DIN",        Pins("R15")),
+        Subsignal("nSYNC",      Pins("J15")),
+        IOStandard("LVCMOS15"),
+    ),
+
     # DDR3 SDRAM
     ("ddram", 0,
         Subsignal("a", Pins(
             "J21 K18 J18 R16 P16 T18 R18 T19",
             "R19 P18 P17 P15 N15"),
-            IOStandard("SSTL135")),
```

Comment:
Some/all of the DDR pins are shared with LVCMOS15 pins. We need to double check what the IO standard for the DDR really is.